client/src/components/recentTaskItem.html

```diff
@@ -0,0 +1,6 @@
+<p ng-repeat="task in recentTaskItem.tasks">
+  <input type="checkbox" id="{{task._id}}" ng-if="task.status === 'done'" checked="checked"/>
+  <input type="checkbox" id="{{task._id}}" ng-if="task.status === 'active'"/>
```

Comment:
we can replace this with one input: 
`<input` type="checkbox" id="{{task._id}}" checked="{{task.status === 'done' ? 'checked' : ''}}"/>`